<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContractorWindow</class>
 <widget class="QDialog" name="ContractorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_header">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Новий контрагент</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tab_widget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>361</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_juridical">
    <attribute name="title">
     <string>Юридична особа</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>136</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="grid_jur">
      <item row="0" column="0">
       <widget class="QLabel" name="label_name_jur">
        <property name="text">
         <string>Назва компанiї</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name_jur"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_code_jur">
        <property name="text">
         <string>Iдентифiкацiйний код</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_code_jur"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_phone_jur">
        <property name="text">
         <string>Номер телефону</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_phone_jur"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_address_jur">
        <property name="text">
         <string>Адреса</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_address_jur"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_physical">
    <attribute name="title">
     <string>Фiзична особа</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>239</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="grid_phy">
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_code_phy"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_phone_phy">
        <property name="text">
         <string>Номер телефону</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_phone_phy"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_surname_phy"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_pathronymic_phy">
        <property name="text">
         <string>По батьковi</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name_phy"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_code_phy">
        <property name="text">
         <string>Iдентифiкацiйний код</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_address_phy">
        <property name="text">
         <string>Адреса</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_address_phy"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_name_phy">
        <property name="text">
         <string>Iм'я</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_surname_phy">
        <property name="text">
         <string>Прiзвище</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_pathronymic_phy"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_birth_phy">
        <property name="text">
         <string>Дата народження</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="dateEdit_birth_phy">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>dd.MM.yyyy</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ContractorWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ContractorWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
